aerospike/content.md

```diff
@@ -1,82 +1,187 @@
-# Aerospike
+# Aerospike Enterprise Edition Docker Image
 
-Aerospike is an open source distributed database. Aerospike is built on a "shared nothing" architecture designed to reliably store terabytes of data with automatic fail-over, replication and cross data-center synchronization.
+## What is Aerospike?
+
+[Aerospike](http://aerospike.com) is a distributed NoSQL database purposefully designed for high performance web scale applications. Aerospike supports key-value and document data models, and has multiple data types including List, Map, HyperLogLog, GeoJSON, and Blob. Aerospike's patented hybrid memory architecture delivers predictable high performance at scale and high data density per node.
 
 %%LOGO%%
 
-Documentation for Aerospike is available at [http://aerospike.com/docs](https://www.aerospike.com/docs).
+-	Getting Started
+	-	Running a node with a feature key file in a mapped directory
+	-	Running a node with a feature key file in an environment variable
+-	Advanced Configuration
+	-	Injecting configuration parameters
+		-	List of template variables
+	-	Using a custom configuration file
+	-	Persistent Data Directory
+	-	Block Storage
+	-	Persistent Lua Cache
+-	Clustering
+	-	Configuring the node's access address
+	-	Mesh Clustering
+-	Image Versions
+-	Reporting Issues
+-	License
+
+## Getting Started
+
+Aerospike Enterprise Edition requires a feature key file to start and to ungate certain features in the database, such as compression. Enterprise customers can use their production or development keys.
 
-# Using this Image
+Anyone can [sign up](https://www.aerospike.com/lp/try-now/) to get an evaluation feature key file for a full-featured, single-node Aerospike Enterprise Edition.
 
-The following will run `asd` with all the exposed ports forwarded to the host machine.
+### Running a node with a feature key file in a mapped directory
 
 ```console
-$ docker run -d --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -p 3003:3003 %%IMAGE%%
+docker run -tid -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 aerospike/aerospike-server-enterprise
```

Comment:
This should reference the `ee` image, right?   I don't think `-ti` should be added absolutely unless it is necessary (and I haven't seen any software that requires a tty unless it is interactive like a shell).

(Same changes should be applied in a few other places below.)

```suggestion
docker run -d -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 %%IMAGE%%:ee-5.5.0.9
```

Though maybe more generic to not reference a specific version:

```suggestion
docker run -d -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 %%IMAGE%%:ee-[version]
```

Reply:
good points on the identifying which env are EE vs. CE and updating docker run and using ee tag with [version] placeholder.  Circling back internally on this.